ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ContinuousPlannerSchedulingTask.java

```diff
@@ -46,321 +36,118 @@
  */
 public class ContinuousPlannerSchedulingTask
 {
-   /**
-    * This limits the number of steps in a session, will cause the state machine to stop
-    */
-   private final static int MAX_STEPS_PER_SESSION = 50;
-
    /**
     * This is the delay between each tick of the state machine. Set based on perception update rate.
     */
    private final static long CONTINUOUS_PLANNING_DELAY_MS = 16;
 
-   private ContinuousWalkingState state = ContinuousWalkingState.NOT_STARTED;
-
-   private enum ContinuousWalkingState
-   {
-      NOT_STARTED, READY_TO_PLAN, NEED_TO_REPLAN, PLAN_AVAILABLE, WAITING_TO_LAND, PAUSED
-   }
-
    protected final ScheduledExecutorService executorService = ExecutorServiceTools.newScheduledThreadPool(1,
                                                                                                           getClass(),
                                                                                                           ExecutorServiceTools.ExceptionHandling.CATCH_AND_REPORT);
 
-   private final AtomicReference<FootstepStatusMessage> footstepStatusMessage = new AtomicReference<>(new FootstepStatusMessage());
-   private final AtomicReference<ContinuousWalkingCommandMessage> commandMessage = new AtomicReference<>(new ContinuousWalkingCommandMessage());
-   private final ROS2Topic controllerFootstepDataTopic;
-   private final ROS2PublisherBasics<PauseWalkingMessage> pauseWalkingPublisher;
-   private final ROS2PublisherMap publisherMap;
    private final TerrainPlanningDebugger debugger;
    private TerrainMapData terrainMap;
 
-   private final ContinuousPlannerStatistics statistics;
-   private final ContinuousHikingParameters parameters;
-   private final FootstepPoseHeuristicChecker stepChecker;
-
    private final ContinuousPlanner continuousPlanner;
-   private final HumanoidReferenceFrames referenceFrames;
 
-   private String message = "";
-   private int stepCounter = 0;
-   private int controllerQueueSize = 0;
-   private List<QueuedFootstepStatusMessage> controllerQueue;
+   public StateMachine<ContinuousHikingState, State> stateMachine;
+   public static YoEnum<ContinuousHikingState> continuousHikingMode;
```

Comment:
this shouldn't be static. It shouldn't exist, but if it did, it shouldn't be static.